phpBB/includes/ucp/ucp_pm_compose.php

```diff
@@ -978,7 +978,26 @@ function compose_pm($id, $mode, $action, $user_folders = array())
 			censor_text($message_parser->message),
 			$quote_attributes
 		);
-		$message_parser->message = $message_link . $quote_text . "\n\n";
+		if ($bbcode_status)
+		{
+			$message_parser->message = $message_link . $quote_text . "\n\n";
```

Comment:
It's better to include the definition part for `$quote_text` inside this branch of the condition.